src/components/member-profile/active-task/index.js

```diff
@@ -40,7 +55,10 @@ const ActiveTask = ({ taskDetails }) => {
         <div className={classNames.progressBar}>
           <div
             className={showProgressIndicator}
-            style={{ width: `${percentCompleted}%` }}
+            style={{
+              width: `${percentCompleted}%`,
+              backgroundColor: `${handleProgressBarColor}`,
```

Comment:
will this work? you are just putting the function value in the background color and not even calling it

Reply:
Kindly verify the new PR.